pkgs/by-name/mc/mcpat/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mcpat";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "HewlettPackard";
+    repo = "mcpat";
+    tag = "v${version}";
+    hash = "sha256-sr7H2vBOTyI59d3itVNqRVy1fR/83ZrTGl5s4I+g0Tw=";
+  };
+
+  patches = [ ./64bit.patch ];
+
+  buildPhase = ''
```

```suggestion
  buildPhase = ''
```
add preBuild postBuild